src/shapes/Arrow.js

```diff
@@ -66,7 +66,21 @@
         ctx.closePath();
         ctx.restore();
       }
+
+      this._fillArrowHead(ctx);
+    },
+
+    // removing dash from arrow-head only, but putting it back to the
+    // intial value after render
+    _fillArrowHead: function(ctx) {
+      var isDashEnabled = this.dashEnabled();
+      if (isDashEnabled) {
+        this.dashEnabled(false);
```

Comment:
I think it is better to avoid such changing because it is triggering attribute changing events.

Reply:
Probably direct changing of `attrs.dashEnabled` will be better. But I am not sure yet.